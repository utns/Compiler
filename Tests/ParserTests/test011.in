type 
    point = record
        x, y: float;
    end;
    
var
    a: array[1..5] of point;
    
begin
    a[1].x := 10.0; 
end.
