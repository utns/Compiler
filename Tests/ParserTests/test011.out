begin
10.000000
:=
     ____x
____.
          ____1
     ____[
          ____a
end
